app/scripts/modules/core/cluster/filter/clusterFilter.service.spec.js

```diff
@@ -769,8 +769,8 @@ describe('Service: clusterFilterService', function () {
       application.clusters = clusterService.createServerGroupClusters(application.serverGroups.data);
       service.updateClusterGroups(application);
       $timeout.flush();
-      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[0]).toBe(this.serverGroup000);
-      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[1]).toBe(this.serverGroup001);
+      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[0]).toEqual(this.serverGroup000);
+      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[1]).toEqual(this.serverGroup001);
```

Comment:
Here:
https://github.com/lwander/deck/blob/kubernetes-job-details/app/scripts/modules/core/cluster/filter/clusterFilter.service.spec.js#L578

Add:
```javascript
category: 'serverGroup',
```
`toBe` should work.